paasta_tools/check_chronos_jobs.py

```diff
@@ -34,6 +34,16 @@ def parse_args():
     return args
 
 
+def _guess_realert_every(chronos_job_config):
+    """Returns the job interval (in minutes) or None if the job has no interval.
+
+    As we run check_chronos_jobs once a minute, we are going to skip
+    all re-alert attempts until the next scheduled run of the job.
+    """
+    interval = chronos_job_config.get_schedule_interval_in_seconds()
+    return (interval // 60) if interval is not None else None
```

Comment:
Max. The reasoning here is that we don't want a batch that runs once per month to only alert once per month. (too easy to forget)